portfolio/src/main/webapp/index.html

```diff
@@ -82,6 +88,30 @@ <h2>My Works</h2>
           </li>
       </ul>
       </div>
+      <div id="contact">
+          <h2>Contact Me</h2>
+
+          <p>
+              We can talk about ANYTHING! Feel free to just reach out.
+          </p>
+
+           <form action ="/form-handler" method="POST">
+                <p>Enter your name:</p>
+                <input class = "input" name="name"/>
```

Comment:
```suggestion
                <input class="input" name="name"/>
```

Please try to keep spacing consistent.